x-pack/legacy/plugins/siem/public/components/news_feed/helpers.ts

```diff
@@ -90,15 +90,11 @@ export const fetchNews = async ({
 }: {
   newsFeedUrl: string;
 }): Promise<RawNewsApiResponse> => {
-  const response = await fetch(newsFeedUrl, {
-    credentials: 'omit',
+  return KibanaServices.get().http.fetch<RawNewsApiResponse>(newsFeedUrl, {
     method: 'GET',
+    credentials: 'omit',
```

Comment:
Scary that I can remove the `credentials: 'omit'` and no unit test catches that mistake. But it does not have the `include` with it so you're probably safe unless someone added all those lines of code here.